langchain/src/document_loaders/web/browserbase.ts

```diff
@@ -0,0 +1,83 @@
+import { Document } from "@langchain/core/documents";
+import type { BrowserbaseLoadOptions } from "@browserbasehq/sdk";
+import { BaseDocumentLoader } from "../base.js";
+import type { DocumentLoader } from "../base.js";
+
+type BrowserbaseLoaderOptions = BrowserbaseLoadOptions & {
```

Comment:
For types which "extend" other types, it's better to use `interface`
```suggestion
interface BrowserbaseLoaderOptions extends BrowserbaseLoadOptions {
```